pyscriptjs/tests/unit/pyconfig.test.ts

```diff
@@ -135,4 +136,38 @@ describe('PyConfig', () => {
         instance.setAttribute("src", "/covfefe.json");
         instance.connectedCallback();
     });
+    it('connectedCallback should call loadRuntimes', async () => {
+        const mockedMethod = jest.fn()
+        instance.loadRuntimes = mockedMethod;
+
+        instance.connectedCallback();
+
+        expect(mockedMethod).toHaveBeenCalled()
+    })
+
+    it('confirm connectedCallback happy path', async () => {
+        const mockedMethod = jest.fn()
+        instance.loadRuntimes = mockedMethod;
+        instance.innerHTML = "test"
+
+        instance.connectedCallback();
+
+        expect(instance.code).toBe("test")
+        expect(instance.values["0"]).toBe("test")
+    })
+
+    it('log should add new message to the page', async () => {
+        // details are undefined, so let's create a div for it
+        instance.details = document.createElement("div")
+        instance.log("this is a log")
+
+        // @ts-ignore: typescript complains about accessing innerText
+        expect(instance.details.childNodes[0].innerText).toBe("this is a log")
+    })
+
+    it('confirm that calling close removes element', async () => {
```

Comment:
this checks for calling of `close` and not really removal I guess?

Reply:
Right, I tried a few things but couldn't really figure out how to test that the element was properly removed so I thought maybe mocking and asserting that `remove` was called was a workaround it

Reply:
Did you try something like `expect(element).not.toBeInTheDocument()`?

Reply:
Yeah something like that, jest doesn't have `toBeInTheDocument`, I tried to poke at the document itself but no luck 😞

Reply:
I guess we can just change the description then and we should be good to go!

Reply:
Done thank you 😄